docs/kratos/social-signin/30_apple.mdx

```diff
@@ -70,7 +93,7 @@ Follow these steps to add Apple as a social sign-in provider to your project usi
 
 Follow these steps to add Apple as a social sign-in provider to your project using the Ory CLI:
 
-1. Using an Apple Developer Account, create an app, a service, and a private key.
+1. Using an Apple Developer Account, create an app, a service, and a private key. Check the Ory Console guide for the concrete steps.
```

Comment:
not a blocker IMO but not ideal to be honest. 

we maybe need to refactor the whole section here - its really a pita to edit these and we get more out of sync with every edit. 

then again its not really broken (yet)

Reply:
Yeah, we can copy paste it over. But I was a bit skeptical over how to do it. Maybe we just put it in the sections before or link to a section? Suggestions welcomed.

Reply:
I think it is fine like this, its a lot of "busy" work to include it in two sections IMO and the structure makes my head hurt. 
I just wanted to comment that we can maybe find a better layout for these guides altogether.

I am thinking 
- howto that is specific to the provider
- and then a short description how you do it in CLI and console that we can copy to all providers
- if needed: a section that explains some special things about this particular provider

This way you only need the steps once, and easy to keep in sync and add new providers.
 
Its probably a couple hours work to update all the guides for this, so I would not put it super high priority but can be done when we clean up the docs a bit, 
WDYT?